site/book/01-getting-started/02-quickstart.md

```diff
@@ -0,0 +1,103 @@
+Let's jump to an example that demonstrates a typical kpt workflow and quickly
+introduce important concepts and features. Following chapters will cover these concepts in detail.
+
+In this example, you are going to configure and deploy Apache Cassandra to a Kubernetes cluster.
+
+## Fetch the package
+
+First, let's fetch the Cassandra package from Git to your local filesystem:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt/package-examples/cassandra@next cassandra
+```
+
+kpt is fully integrated with Git and enables forking, rebasing and versioning a package of
+configuration using the underlying Git version control system. Since kpt is Git-native, any
+existing directory in Git containing Kubernetes resources is also a valid kpt package!
```

Comment:
Might be splitting hairs here, but is it a valid kpt package (since that would imply kpt packages doesn't need to have a Kptfile), or should we frame this as any directory in git with KRM resources can be fetched into a kpt package?